.github/workflows/docs-workflow.yml

```diff
@@ -95,3 +95,21 @@ jobs:
         run: |
           git status
           git diff-index --quiet HEAD
+
+  # Fail if there are de-referenced links in the repository
+  check-de-referenced-links:
+    needs: path_filter
+    if: ${{ needs.path_filter.outputs.should_not_skip == 'true' }}
+    name: Check for de-referenced links
+    runs-on: ubuntu-20.04
```

Comment:
Nit: Can we use 22.04?

Reply:
From what I can tell almost all workflows currently run on ubuntu-20.04. Migrating them to ubuntu-22.04 (or ubuntu-latest) could be another PR. Within the scope of this PR I would rather stay consistent with what is already there.